models/ninjas.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const mongoose = require('mongoose');
+const Schema = mongoose.schema;
+
+const ninjaItem = Schema({
+  ninja:{type: String, required: true},
+  clan:{ type: String, max: 1048},
+  weapons:{ type: String, max: 1048},
+  date:{type: Date, default: Date.now},
+});
+
+module.exports = mongoose.model('list', ninjaItem);
```

Comment:
Generally, you'll want to match schema names to what will be referred to in mongo. Mostly for readability, but ultimately user preference.